
Loading dataset...
Loaded data : 62235
batch_size = 128
learning rate = 0.0001
train_size = 49788
test_size = 12447
Data load complete!
Initializing model....
Initialize complete!!
Epoch 1
step : 6/389, train_loss : 69735.234375, test_loss : 71556.570312555
CVAE_main.py:154: MatplotlibDeprecationWarning: savefig() got unexpected keyword argument "figsize" which is no longer supported as of 3.3 and will become an error in 3.6
  plt.savefig('recon.png', figsize=(16, 16))
CVAE_main.py:157: MatplotlibDeprecationWarning: savefig() got unexpected keyword argument "figsize" which is no longer supported as of 3.3 and will become an error in 3.6

























epoch 1 - average loss - train : 48118.84375, test : 47947.953125640625
Epoch 2
























epoch 2 - average loss - train : 43935.3671875, test : 43849.6132812575
Epoch 3
























epoch 3 - average loss - train : 43761.390625, test : 43680.53906257555
Epoch 4
























epoch 4 - average loss - train : 43639.91015625, test : 43562.886718755
Epoch 5

























epoch 5 - average loss - train : 43448.05859375, test : 43374.738281255
Epoch 6


























epoch 6 - average loss - train : nan, test : nannnloss : 45741.75390625
Epoch 7


























epoch 7 - average loss - train : nan, test : nann
Epoch 8

























epoch 8 - average loss - train : nan, test : nann
Epoch 9



























epoch 9 - average loss - train : nan, test : nann
Epoch 10

























epoch 10 - average loss - train : nan, test : nan
Epoch 11


























epoch 11 - average loss - train : nan, test : nan
Epoch 12


























epoch 12 - average loss - train : nan, test : nan
Epoch 13

















step : 242/389, train_loss : nan, test_loss : nan
Traceback (most recent call last):
  File "CVAE_main.py", line 143, in <module>
    state, train_loss = train_step(state, x, y, rng)
  File "/usr/local/lib/python3.8/dist-packages/flax/core/frozen_dict.py", line 159, in tree_unflatten
    @classmethod
KeyboardInterrupt
Traceback (most recent call last):
  File "CVAE_main.py", line 143, in <module>
    state, train_loss = train_step(state, x, y, rng)
  File "/usr/local/lib/python3.8/dist-packages/flax/core/frozen_dict.py", line 159, in tree_unflatten
    @classmethod
KeyboardInterrupt